admin/acceptance.py

```diff
@@ -636,12 +635,15 @@ def _create_and_provision(self, reactor, name, retries=10):
         :return: Deferred that fires with the created node when
             the provisioning is completed.
         """
+        print "re-creating and re-trying node", name
```

Comment:
That makes sense but `print`s are now pervasive in that code. I've created an issue: https://clusterhq.atlassian.net/browse/FLOC-4096